src/components/invitation/Invitation.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { Box, Grid, Typography } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import theme from '../../styles/createTheme'
+
+const Invitation = () => {
+  const { classes } = useStyles()
+
+  return (
+    <Grid container spacing={3}>
+      <Grid item xs={12}>
+        <Typography variant="h1" className={classes.title}>
+          Invite
+        </Typography>
+        <Box
+          className={classes.imageWrapper}
+          component="img"
+          src="/img/invite/qr-invite.svg"
+        ></Box>
+        <Typography variant="body2" className={classes.description}>
+          Feel free to invite someone, who can be interesting in finding friends
+        </Typography>
+      </Grid>
+    </Grid>
+  )
+}
+
+export default Invitation
+
+const useStyles = makeStyles()(() => {
+  return {
+    title: {
+      textAlign: 'center',
+      paddingBottom: '50px',
```

Comment:
all measures with accurate numbers can be written without `px` and strings, just numbers, e.g. 
`paddingBottom: 50`

Reply:
This commit is from another branch: [nearby-no-users]. The mentioned branch should be merged into the main branch sooner than this one. It's still on hold, could you please review [nearby-no-users] again?"